src/classes/UTIL_JavaScriptSanitizer.cls

```diff
@@ -96,10 +96,8 @@ public with sharing class UTIL_JavaScriptSanitizer {
     private String replaceCommasWithSplitMarkers(String toParse) {
         Boolean inQuotes = false;
         String openQuoteChar;
-        Integer i = 0;
-
         String substitutedValue = '';
-        do {
+        for ( Integer i = 0; i < toParse.length(); i++ ){
```

Comment:
Should this make that if statement no longer necessary?

Reply:
P.S. Extra spaces after the open ( and before the close and missing after the open :)

Reply:
ah good catch! It certainly is Friday